workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinker.cs

```diff
@@ -8,13 +8,13 @@ namespace Improbable.Gdk.Core
     public class EntityGameObjectLinker
     {
         private readonly World world;
-        private MutableView view;
+        private readonly ILogDispatcher logDispatcher;
         private readonly HashSet<Type> gameObjectComponentTypes = new HashSet<Type>();
 
-        public EntityGameObjectLinker(World world, MutableView view)
+        public EntityGameObjectLinker(World world)
         {
             this.world = world;
-            this.view = view;
+            logDispatcher = new ForwardingDispatcher();
```

Comment:
Why is a core system creating its own dispatcher? Isn't the thought that a user can easily define which dispatcher they want to use?

Reply:
moved out :)